libs/password-complexity/index.js

```diff
@@ -0,0 +1,41 @@
+const validators = [
+  {
+    name: 'Password must be at least 15 characters',
+    check: function checkLength(password) {
+      return password.length >= 15;
+    }
+  },
+  {
+    name: 'Password must contain a combination of lowercase letters, uppercase letters, numbers, and special characters',
```

Comment:
We may want to break this into 4 separate checks:
- Contains capital letter(s)
- Contains lowercase letter(s)
- Contains number(s)
- Contains special character(s)

Reply:
I've tried this and it seems to be too many fields to fit here

![image](https://user-images.githubusercontent.com/66680985/134068874-a5420127-bfe9-4be4-ba96-b5016718fd3c.png)